configs/README.md

```diff
@@ -1,149 +1,78 @@
 # Experiment Config Demystified
 
-An experiment config is a `.json` file consisting of the following fields:
+## Experiment Config and Task Concept
 
-- `collect_results_path`:  The directory path where the results produced by the experiment should be copied,
-- `ignore_stop_order`: True/False depending on whether the stop order specified in tasks' configs should be/should not be ignored when stopping the experiment,
-- `stop_after`: The time, in seconds, after which the experiment should be stopped,
-- `tasks`: The list of tasks to run within the experiment and their configs.
+Experiment config is a `.json` file consisting of the following fields:
```

Comment:
@mbakholdina Is "Experiment config" the name of the file?

Reply:
Here it's in general any config file.